c2cgeoportal/scaffolds/create/apache/application.wsgi.mako

```diff
@@ -1,15 +1,28 @@
 import site
 import sys
 import re
+import os
+from logging.config import fileConfig
 
+% if docker == 'TRUE':
+root = "/app"
+% else:
 site.addsitedir("${python_path}")
+root = "${directory}"
+% endif
+
 
 # Remove site packages
 regex = re.compile("^/usr/lib/python.\../dist-packages$")
 sys.path = [p for p in sys.path if regex.match(p) is None]
 
-from pyramid.paster import get_app, setup_logging
+from pyramid.paster import get_app
```

Comment:
Humm, actually, that works with the import at the top only in docker. For a normal install, the import must be after the addsitedir call and sys.path hacking. Otherwise we have this error:
```
Traceback (most recent call last):
   File "/tmp/travis/testgeomapfish/apache/application.wsgi", line 5, in <module>
     from pyramid.paster import get_app
ImportError: No module named pyramid.paster
```